app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,7 @@
 class PostsController < ApplicationController
   def index
```

Comment:
- [ ] To activate the CanCan gem abilities for a user it should be invoked in the posts controller using `load_and_authorize_resource`